src/Features/LanguageServer/Protocol/Features/Options/WorkspaceConfigurationOptionsStorage.cs

```diff
@@ -50,5 +50,5 @@ public static WorkspaceConfigurationOptions GetWorkspaceConfigurationOptions(thi
             SourceGeneratorExecutionPreferenceUtilities.GetEditorConfigString));
 
     public static readonly Option2<bool> SourceGeneratorExecutionBalancedFeatureFlag = new(
-        "dotnet_source_generator_execution_balanced_feature_flag", false);
+        "dotnet_source_generator_execution_balanced_feature_flag", true);
```

Comment:
note: we still have the feature flag.  So we still have control-tower support to flip this back to 'false' if we run into any problems with this in the wild.

Reply:
done.